dpdispatcher/openapi.py

```diff
@@ -0,0 +1,189 @@
+import os
+import shutil
+import time
+
+from bohriumsdk.client import Client
+from bohriumsdk.job import Job
+from bohriumsdk.storage import Storage
+from bohriumsdk.util import Util
+
+from dpdispatcher import dlog
+from dpdispatcher.JobStatus import JobStatus
+from dpdispatcher.machine import Machine
+
+shell_script_header_template = """
+#!/bin/bash -l
+"""
+
+
+class OpenAPI(Machine):
```

Comment:
Documentation should be added for a new `Machine` [here](https://github.com/deepmodeling/dpdispatcher/blob/8ee05a704815745bfb32a91493a772df9d29b6a6/doc/batch.md).